python/ray/serve/tests/test_schema.py

```diff
@@ -478,66 +478,6 @@ def test_serve_application_invalid_import_path(self, path):
         with pytest.raises(ValidationError):
             ServeApplicationSchema.parse_obj(serve_application_schema)
 
-    def test_serve_application_aliasing(self):
```

Comment:
@shrekris-anyscale anywhere else do we need to change?

Reply:
No, I believe everything is updated.